packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -216,7 +217,7 @@ export class OdspDelayLoadedDeltaStream {
 		await new Promise<void>((resolve, reject) => {
```

Comment:
Can you please glance at joinSessionCore() where this function is called?
We log error only if it's non-retryable error. But we never retry. That's probably fine (we will recover by reconnecting), but it feels like if that's design, we should log errors always. If we do want to retry, they we have to respect retryAfter property on error returned and also have some back-off logic (i.e. ideally use some helper function like runWithRetry), and I'm not sure it's worth it. But we need to log these errors at least.